code/datums/components/footstep.dm

```diff
@@ -38,13 +38,14 @@
 			footstep_sounds = 'sound/effects/footstep/slime1.ogg'
 		if(FOOTSTEP_OBJ_MACHINE)
 			footstep_sounds = 'sound/effects/bang.ogg'
-			RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/play_simplestep_machine) //Note that this doesn't get called for humans.
+			RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/play_simplestep_machine)
```

Comment:
is there a reason to remove the comment?

Reply:
That would be ***another*** leftover change which I forgot to undo. In this case I got rid of them in the first commit since the initial method removed `FOOTSTEP_MOB_HUMAN` entirely, so the comments didn't apply anymore.

Since this seems to keep coming up I'll do a quick pass of the whole PR to make sure everything's where it's meant to be.
~~(Would you believe that I already went over it twice before?)~~